<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proveedores</class>
 <widget class="QMainWindow" name="Proveedores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>348</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>231</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Proveedores</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BIENVENIDOS A PROVEEDORES</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblProveedores">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>611</width>
      <height>221</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <column>
     <property name="text">
      <string>CODIGO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CONTACTO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DIRECCIÃ“N</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRECIO DISTINTIVO</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAgregarP">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>AGREGAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/agregar-usuario.png</normaloff>../assets/images inventario/agregar-usuario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminarP">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 123, 123);</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/borrar.png</normaloff>../assets/images inventario/borrar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditarP">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>EDITAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/editar-codigo.png</normaloff>../assets/images inventario/editar-codigo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegresarP">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/images inventario/flecha-hacia-atras.png</normaloff>../assets/images inventario/flecha-hacia-atras.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
